databricks-sdk-java/src/main/java/com/databricks/sdk/core/BricksCliCredentialsProvider.java

```diff
@@ -9,17 +9,17 @@ public class BricksCliCredentialsProvider implements CredentialsProvider {
 
   private static final Logger LOG = LoggerFactory.getLogger(BricksCliCredentialsProvider.class);
 
-  public static final String BRICKS_CLI = "bricks-cli";
+  public static final String BRICKS_CLI = "databricks-cli";
```

Comment:
This change doesn't affect which binary is being used but we should do it for consistency.